crates/driver/openapi.yml

```diff
@@ -279,94 +337,110 @@ components:
         auctionId:
           description: Id of the auction that should be executed.
           type: integer
-        transactionIdentifier:
-          description: |
-            Extra calldata that is appended to the settlement contract invocation by the Solver to
-            identify the transaction.
-
-            Hex encoded bytes. No fixed length.
-          type: string
-          example: "0x00112233"
+    RevealedResponse:
+      description: Response of the reveal endpoint.
+      type: object
+      properties:
+        calldata:
+          $ref: "#/components/schemas/Calldata"
     SettleResponse:
-      description: Response of the execute endpoint.
+      description: Response of the settle endpoint.
       type: object
       properties:
-        clearingPrices:
+        calldata:
+          $ref: "#/components/schemas/Calldata"
+        txHash:
           description: |
-            Mapping of hex token address to price.
+            The hash of the transaction that executed the solution.
 
-            The prices of tokens for settled user orders as passed to the settlement contract.
-          type: object
-          additionalProperties:
-            $ref: "#/components/schemas/BigUint"
-        trades:
-          description: executed orders
-          type: array
-          items:
-            type: object
-            properties:
-              uid:
-                $ref: "#/components/schemas/OrderUID"
-              executedAmount:
-                $ref: "#/components/schemas/TokenAmount"
-        internalizedInteractions:
-          type: array
-          items:
-            type: object
-            description: |
-              An internalized interaction is an interaction that could have executed on chain but
-              was instead handled through the settlement contract's buffers to save gas. The
-              information included here allows reconstructing the interaction so that it can be
-              checked for validity.
-            properties:
-              calldata:
-                description: hex encoded
-                type: string
-                example: "0x01"
-              inputs:
-                description: Mapping of hex token address to amount.
-                type: object
-                additionalProperties:
-                  $ref: "#/components/schemas/TokenAmount"
-              outputs:
-                description: Mapping of token address to amount.
-                type: object
-                additionalProperties:
-                  $ref: "#/components/schemas/TokenAmount"
-        calldata:
-          description: hex encoded
+            This is the hash that should be used to submit the solution.
           type: string
-          example: "0x01"
-        signature:
-          description: |
-            Signature confirming that the Solver promised to execute this auction.
+          example: "0x2fa8a928efdd11fcf09e196dfa6e3ac4884967a9409ecc1b854ed2eabe4839e2"
```

Comment:
I think `calldata` is an object like:
```json
{
    "internalized": "0xabcd",
     "uninternalized": "0xabcdefgh",
}
```

Reply:
This was moved and is now explaining tx_hash which is just that.